### DISCUSSION TITLE
[Workshop] Cite.Software Workshop Notes and Curated Resources

### DISCUSSION THREAD

**Discussion by malvikasharan** [the-turing-way/the-turing-way#4255] [General]
### Summary

We will be participating in Cite.Software workshop before the Research Software Funders Forum.
The proposal was put together by Chris Erdmann, with contributions from Maria Cruz, Maaike de Jong, Jason Maassen, Lauren Cadwallader, Fares Dhane and Malvika Sharan

- Audience: Researchers/Authors
- Challenge: Guidance on research software sharing is scattered across many libraries, journals, research groups, and organizations' websites. The guidance varies across these websites and does not often address researchers' needs for clear instruction. Researchers/authors are left without a common reference point.
- Solution: Funders drive the adoption of a common research software preservation and citation guidance website called Cite.Software (domain secured) for the research community. Content for the site will be derived from the [Turing Way’s Handbook](https://the-turing-way.netlify.app/reproducible-research/reproducible-research) and The Netherlands eScience Center’s[ FAIR software recommendations](https://fair-software.eu/). Development of the site’s instructions for researchers will be led by Turing Way and The Netherlands eScience Center. They in turn will work with key stakeholders involved with research software preservation and citation to further develop the content. The site will be hosted via GitHub (or similar) to foster transparent, collaborative work with stakeholders and ultimately the community.

**Agenda**
  - Cite.Software Overview (Chris) - ~30 min / Q&A
  - Software at PLoS/Survey (Lauren) - ~15 min / Q&A
  - Break 10 min
  - Turing Way Software Content/Approach (Malvika) - ~ 15 min / Q&A
  - eScience Center & the Research Software Directory (Maaike) - ~ 15 min / Q&A
  - Software Management Plans (Maaike for Maria) - ~ 15 min / Q&A
  - Discussion & Next Steps

**Reference**: 
- Software Management Plans. https://www.software.ac.uk/software-management-plans 
- @c-martinez, Martinez Lavanchy, P., Sesink, L., Olivier, B. G., Meakin, J., de Jong, M., & Cruz, M. (2023). Practical guide to Software Management Plans. Zenodo. doi: 10.5281/zenodo.7589725

### What needs to be done?

- [ ] Curate chapters for display in the curated pathway
- [ ] Invite feedback - share feedback process

### Who can help?

* All folks attending the workshop
* Anyone who is interested in this topic after the workshop

---

### Updates

* The workshop is taking place on 18th Sep

**Comment by malvikasharan** [comment]
Chapters selection that will be displayed:
- Introduction to research software
- Data management plan --> Software
- GitHub
- License
- Cff Citation
- Documentation

Create one for RSE that includes
- Software practices like code testing, code review, environment, workflow etc.

**Comment by c-martinez** [comment]
Other sections that relate to RSE:

https://the-turing-way.netlify.app/reproducible-research/testing
https://the-turing-way.netlify.app/reproducible-research/reviewing
https://the-turing-way.netlify.app/reproducible-research/renv
https://the-turing-way.netlify.app/reproducible-research/ci
https://the-turing-way.netlify.app/reproducible-research/code-quality

**Comment by malvikasharan** [comment]
Thank you @c-martinez -- I am going to add these to the curation, which will is currently being displayed as the 4th pathway here: https://deploy-preview-3277--the-turing-way.netlify.app/#different-pathways

**Comment by c-martinez** [comment]
Good luck with the Workshop! I am not sure if I will be able to join, but I am sure it will be great!

**Comment by jcolomb** [comment]
Would it make sense to include the RDA to such an initiative, maybe creating a new group ?

https://www.rd-alliance.org/groups/complex-citations-working-group
https://www.rd-alliance.org/groups/fair-research-software-fair4rs-wg

**Comment by malvikasharan** [comment]
Hi folks, in this issue. I hope you are well!

If you hadn't noticed, a pathway with the initial set of chapters was added here: https://book.the-turing-way.org/pathways/software-citation. Special shout-out to @libcce and @c-martinez for their thought leadership on Cite.Software project. 🎉 

We had applied for some funding to support the documentation of additional chapters, but were not successful, and then I think excitement around this work faded as we became busier.

BUT, I haven't forgotten this, and as part of Book Dash this time, we wrote a chapter describing the Pathway feature, highlighting an example from this workshop: https://book.the-turing-way.org/community-handbook/pathways/pathways-example#ch-pathways-example-citation. I hope this can nudge more people to both contribute to this pathway and create new ones.

Please do re-engage if you have capacity, existing resources to add or simply use resources from The Turing Way.

I will close this issue next week if there is no follow-up. ✨

**Comment by jcolomb** [comment]
I have no capacity at the moment, but this sounds as an interesting pathway.

I think it needs some content to work on citable software outside of Github (it is there https://book.the-turing-way.org/communication/citable/citable-cite#cm-citable-cite-software but it would need to show in the table of content, took me 2min to find again, while I knew it was there somewhere ;) ).

**Comment by libcce** [comment]
Hi @malvikasharan can you give us another 2 weeks before closing this issue? @almanilsson is reaching out to Fotis/EVERSE to see if their RSQKIT responds to this issue or whether we should just set up cite.software and invite people. @c-martinez mentioned that maybe we should give this a try.

**Comment by libcce** [comment]
@malvikasharan both @almanilsson and I believe you can close this issue now. We will follow up with EVERSE RSQKIT but the current thinking is to set up cite.software, with the material at hand, and encourage collaboration. Of course, much of what we have leverages material in Turing Way (so that may be a separate thread). But for now we can close this issue at least.

**Comment by malvikasharan** [comment]
@libcce, Thank you for the update. Super excited to hear the progress from EVERSE!!
This issue can stay open to keep checking in - I will convert this to a discussion now. 💐
